cosmosdb/constants.bal

```diff
@@ -112,7 +112,7 @@ public enum OfferVersion {
     USER_DEFINED = "V2"
 }
 
-# The performance levels for a specific throughput level. They depend on the Cosmos DB region which the container 
+# Performance levels for a specific throughput level. <br/> They depend on the Cosmos DB region which the container 
```

Comment:
Suggestion for the second sentence: "They depend on the Cosmos DB region to which the contain the belongs and on whether the container is single partitioned or multiple partitioned."